main.tf

```diff
@@ -195,9 +195,6 @@ tsm configuration set -k  storage.monitoring.email_enabled -v true
 echo "#TSM configure access to peering proxies"
 tsm configuration set -k wgserver.systeminfo.allow_referrer_ips -v ${var.haproxy_private_ip},${var.haproxy_private_ip2}
 
-echo "#TSM apply pending changes"
-tsm pending-changes apply
-
 echo "#TSM initialise & start server"
 tsm initialize --start-server --request-timeout 1800
```

Comment:
Have you checked that you can remove that "apply pending-changes" _before the init?
Or is this change to do the testing (in NotProd)?

Reply:
I haven't checked that.  The change is to do the testing in NotProd.  
If there is a way to deploy to NotProd from my Branch that would be preferable, as there is a Prod B/G deployment tomorrow and I would need to revert any changes that prove to be non-viable prior to that.